.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+
+python:
+- 3.3.0
```

Comment:
I remember astrodbkit having similar issues with Python 3.3, I think it was some incompatibility with numpy. I wouldn't test against 3.3 anyways, maybe just 2.7, 3.5, and 3.6.